juju/juju
# Cloud facade only returns clouds a user can see

## Description of change

The Cloud(name) and Clouds() API calls on the Clouds facade only return clouds a user has permission to see.

## QA steps

Primarily internal api change.
smoketest - bootstrap and ensure user can add-model etc
